app/gist/controller.js

```diff
@@ -123,6 +189,11 @@ export default Ember.Controller.extend({
     }
     return false;
   },
+
+  getColumnFile(column) {
+    return this.get(`col${column}File`);
```

Comment:
with https://github.com/ember-cli/ember-twiddle/pull/141/files#r37147243 this would likely become:

```js
this.get('column').objectAt(column)
```